<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map</class>
 <widget class="QWidget" name="Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ant Simulation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/assets/ant.ico</normaloff>:/assets/ant.ico</iconset>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1091</width>
     <height>661</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>761</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QLabel" name="labelMapSize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Map size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputX">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>X=100</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAnt">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>141</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/assets/ant.ico</pixmap>
    </property>
   </widget>
   <widget class="QComboBox" name="inputFood">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelObstacle">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Obstacle spawn</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inputObstacle">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>high</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelFood">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Food spawn</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>501</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Select the parameters for the simulation</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelEvaporation">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Evaporation rate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputY">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Y=50</string>
    </property>
   </widget>
   <widget class="QComboBox" name="inputFood_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>72</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Low</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
